posthog/management/commands/create_channel_definitions_file.py

```diff
@@ -186,7 +237,10 @@ def handle_entry(entry):
 
         def sort_key(row):
             name, kind, hostname_type, type_if_paid, type_if_organic = row
-            source_fld = get_fld(name, fail_silently=True, fix_protocol=True)
+            if name and any(name.startswith(s) for s in ("com.", "xyz.", "org.")):
```

Comment:
I didn't add tv.* here, even though tv.twitch is in the list, because tv.* is likely a somewhat common subdomain. It's low stakes as this is just the order of stuff in the generated json file, so 🤷 not worth doing something clever here